### PR TITLE
DOCS-4105 Timeseries Widget Doc Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Refreshes the Timeseries Widget doc page and screenshots, clarifies feature documentation about adding additional Y-Axis lines to the widget graph, and updates some page links.

### Motivation

DOCS-4105 + Hotjar

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/dashboards/widgets/timeseries.md [hunk 1]
```diff
@@ -1,132 +1,142 @@
...
-Add configurable legends on your screenboards by navigating to the legend section in the graph editor and selecting an option.
+To add markers for additional data sets, click **Add Marker** in the **Markers** section.
 
-{{< img src="dashboards/widgets/timeseries/legend-config.jpg" alt="Legend configuration" style="width:100%;" >}}
+1. Select a Line or Range and input a value or a range or values.
+2. In the **Show as** field, select an alerting status/color and choose from a solid, bold, or dashed horizontal line. 
+3. To add a label that displays on the bottom left of the timeseries widget, define a value for the Y-Axis and click the **Label** checkbox. 
 
-Options:
+### Y-Axis controls
```
Comment (urseberry) [inline content/en/dashboards/widgets/timeseries.md:93]: I don't know if you want to take this on now, but all of these options could use some clearer organization. If you look in the app, there are 5 options all grouped together: Event Overlays, Markers, Y-Axis Controls, Legend, and Context Links. These could go together under an H3 heading, maybe something like "Graph options."

Comment (urseberry) [reply content/en/dashboards/widgets/timeseries.md:93]: Also, throughout this doc, "Y-Axis" should be lowercased as "y-axis"

**Comment by omar-nema** [comment]
good to go from my end!

**Review by omar-nema — APPROVED** [review]
Good to go, feedback addressed!

File: content/en/dashboards/widgets/timeseries.md [hunk 2]
```diff
@@ -1,132 +1,142 @@
...
 
-* Automatic (default)
-* Compact
-* Expanded: Configurable columns for value, avg, sum, min, max
-* None
+Y-axis controls are available in the UI and in the JSON editor. You can set the value and type of the y-axis in order to:
 
-Note: For timeboards, legends display automatically when dashboard is set to L or XL.
+* Clip the y-axis to specific ranges.
+* Automatically change y-axis bounds based on a percentage or an absolute value threshold. This threshold can be applied to one of both ends of the graph (lower and upper) in order to remove the "outliers" series.
```
Comment (urseberry) [inline content/en/dashboards/widgets/timeseries.md:98]:
```suggestion
* Automatically change y-axis bounds based on a percentage or an absolute value threshold. This threshold can be applied to one or both ends of the graph (lower and upper) in order to remove the "outliers" series.
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**